components/Alfabet.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { useState } from 'react'
+import { View, Vibration } from 'react-native'
+
+import styled from 'styled-components/native'
+
+const TouchButton = styled.TouchableOpacity`
+  padding: 25px;
+  margin: 0px;
+  width: 250px;
+  borderRadius: 25px;
```

Comment:
We shouldn't use camelCase in styled components, normal kebab-case is ok. So :

```
border-radius
```

instead of 

```
borderRadius
```